### ISSUE TITLE
Multiple RSS feeds

### ISSUE THREAD

**Issue by jhfrench** [sdepold/jquery-rss#23]
It would be very useful if jQuery-RSS supported multiple RSS sources, ie $.rss(["http://feed1.com/rss","https://feed2.org/rss/feed.xml"], {...}.

**Comment by jayanta119** [comment]
It would be very useful if jQuery-RSS supported  pagination   very usefull  ,,,

**Comment by jayanta119** [comment]
Zundrium  

its not working  multiple RSS sources ....

**Comment by jhfrench** [comment]
@jayanta119 it seems to work for me. See http://jsfiddle.net/jhfrench/ffmypq9r/2/

**Comment by jayanta119** [comment]
thanx for sending me a exampl i working @jhfrench

**Comment by sdepold** [comment]
@jhfrench If you'd send me a PR I could potentially add this functionality to the lib.

**Comment by jhfrench** [comment]
@sdepold : I will--but I can't take credit for it. It's @jayanta119's code, even if I lint it and submit it for him.

**Comment by sdepold** [comment]
Ah i see :) anyways a pull request would be welcome

**Comment by Zundrium** [comment]
@jhfrench : Go ahead mate! It's all yours :+1:

**Comment by jhfrench** [comment]
I have incorporated Zundrium's enhancements to support multiple RSS sources, but cannot test due to ["moment" dependency introduced by Ross Dallaire Jan 5, 2015](https://github.com/sdepold/jquery-rss/pull/43); see JS console error at  [http://jsfiddle.net/jhfrench/ffmypq9r/3/](http://jsfiddle.net/jhfrench/ffmypq9r/3/)

**Comment by jhfrench** [comment]
Included moment library to get past that dependency, now getting "Uncaught TypeError: Cannot read property 'onData' of undefined". See http://jsfiddle.net/jhfrench/ffmypq9r/4/

**Comment by jhfrench** [comment]
Got past that error. I'm sure you're happy I'm clogging up your issue feed with my blow-by-blow.

Meanwhile, seems to be working: http://jsfiddle.net/jhfrench/ffmypq9r/5/

For what other use cases should I test it?

**Comment by jhfrench** [comment]
@sdepold  do you have a standard set of use-cases I can use to regression test?

**Comment by jhfrench** [comment]
@sdepold do you have a standard set of use-cases I can use to regression test?

**Comment by sdepold** [comment]
Not really. There are tests in the project that should pass. Other than then I don't have any specific use cases in mind

**Comment by jhfrench** [comment]
Ok...so the next step is for me to issue a PR?

**Comment by sdepold** [comment]
Yessir
Jeromy French <notifications@github.com> schrieb am Mo., 1. Juni 2015 um
15:47:

**Comment by nostalgiaz** [comment]
Hi guys! 
I need this feature too... when do you think you'll able to release it (via bower)?

Thanks again!

**Comment by uudruid74** [comment]
The ability to demand-load additional items becomes much more important once you load multiple feeds at once.   If you want a very easy way to merge your feeds, just use http://www.rssmix.com/  

This will create a custom mash-up of the feeds you select and automatically does its own caching and such.   Very cool and easy.  Limit 1 request per second!

However, like most RSS feeds, I would suggest that a demand-load system only fetch the RSS.XML once and then adjust the output HTML accordingly (see my comment on the offsetStart/End bug).  If you constantly fetch the xml file then you may be cut off from sources that don't want you slamming the server every time someone slides a scroll wheel on your site!  Which is likely what would happen if you used offsetStart/End to demand-load pieces of a feed and then concatenated them with jQuery after the load.

**Comment by sdepold** [comment]
This just landed in vanilla-rss: https://github.com/sdepold/vanilla-rss#multiple-feed-urls

Will update jquery-rss later today.

**Comment by sdepold** [comment]
Just released v4.2.0 of jquery-rss and 1.3.0 of vanilla-rss that support multiple feeds

**Comment by sdepold** [comment]
v4.3.0 is now also supporting ordering of the feeds in case you need this:

```
$("#rss-feeds").rss([
            "https://www.contentful.com/blog/feed.xml",
            "http://www.ebaytechblog.com/feed/"
          ], {
order: 'publishedDate'
})
```
